test/setup/server_setup.go

```diff
@@ -103,6 +103,25 @@ func (s *Server) AdminSigner() crypto.Signer {
 	return s.adminSigner
 }
 
+func (s *Server) QueryClusterStatus(t *testing.T) (*types.GetClusterStatusResponseEnvelope, error) {
+	client, err := s.NewRESTClient(nil)
+	if err != nil {
+		return nil, err
+	}
+
+	query := &types.GetClusterStatusQuery{
+		UserId: s.AdminID(),
```

Comment:
Unlike GetConfig (in a future PR we plan to limit getConfig to admin only), this does not have to be `admin`, can be a regular user `alice`. Let's do it with a regular user if we can.

Reply:
Lets handle this in a separate issue: https://github.com/hyperledger-labs/orion-server/issues/341